chapters/01.Brackets/brackets.jl

```diff
@@ -0,0 +1,1156 @@
+### A Pluto.jl notebook ###
+# v0.16.0
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 9c340f8b-877d-41b9-baca-2698cc2b5988
+using Plots
+
+# ╔═╡ 99adb0e2-1b94-11ec-1351-79707f6016af
+md"""
+# Bracket search
+
+*STMO*
+
+**Michiel Stock**
+"""
+
+# ╔═╡ 6b30fe20-f321-4cb8-a2fc-0a2820d71bc0
+md"As a first introduction to mathematical optimization, we will study two *bracketing methods*. Bracketing methods can be used to minimize scalar function with a single input variable. These algorithms identify an interval $[a, b]$ containing the desired minimum. We will assume that the functions are *unimodal* they contain a single minimum."
+
+# ╔═╡ bdf6c059-db72-4d90-b98f-2e77b7d89f37
+md"""
+## Finding an initial bracket
+
+The following function will generate from an initial $x$ an interval containing a local minimum. It works by moving one of the limits of the interval until the function value at that limit starts to decrease.
```

Comment:
It works by moving one of the limits of the interval until the function value starts to **increase**?